app/assets/javascripts/components/accountLink.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const ACCOUNT_PATHNAME = '/dashboard/account';
+
+export default function AccountLink() {
+  return (
+    <a href={ACCOUNT_PATHNAME}>
```

Comment:
Ah so the accountLink is in the headers so it's outside the react-router scope..